src/contexts/StateContextProvider.tsx

```diff
@@ -419,6 +421,8 @@ const StateContextProvider = ({children}: StateContextProviderProps) => {
             cursor: cursor,
             decoderOptions: getConfig(CONFIG_KEY.DECODER_OPTIONS),
         });
+        setQueryResults(STATE_DEFAULT.queryResults);
+        setQueryProgress(QUERY_PROGRESS_VALUE_MIN);
```

Comment:
similar to the other state variables, can we reset these at the beginning of `loadFile`?